Source/JavaScriptCore/inspector/InspectorBackendDispatcher.h

```diff
@@ -40,6 +40,12 @@ class BackendDispatcher;
 
 typedef String ErrorString;
 
+template <typename T>
+using CommandResponse = Inspector::Protocol::ErrorStringOr<T>;
+
+template <typename T>
+using CommandCallback = Function<void(CommandResponse<T>)>;
```

Comment:
out of curiosity, any reason to not use `CompletionHandler` since we always expect it to be called once?

NIT: i feel like maybe `AsyncCommandCallback` (and `AsyncCommandResponse`) would be a bit better to clarify that it's only used by things marked `"async": true` instead of _all_ commands

Reply:
It has stricter semantics and adopting it everywhere is not always appropriate–and if you mess up it crashes. But now that this refactor is scoped to just WebDriver related code at the moment, it might be workable?